sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/DigitalTwinsAsyncClient.java

```diff
@@ -43,15 +43,14 @@
  * the digital twin models and event routes tied to your Azure Digital Twins instance.
  * </p>
  */
-@ServiceClient(builder = DigitalTwinsClientBuilder.class)
+@ServiceClient(builder = DigitalTwinsClientBuilder.class, isAsync = true)
 public final class DigitalTwinsAsyncClient {
     private static final ClientLogger logger = new ClientLogger(DigitalTwinsAsyncClient.class);
     private static final ObjectMapper mapper = new ObjectMapper();
-    private final DigitalTwinsServiceVersion serviceVersion;
     private final AzureDigitalTwinsAPIImpl protocolLayer;
     private static final Boolean includeModelDefinitionOnGet = true;
 
-    DigitalTwinsAsyncClient(HttpPipeline pipeline, DigitalTwinsServiceVersion serviceVersion, String host) {
+    DigitalTwinsAsyncClient(HttpPipeline pipeline, String host) {
```

Comment:
How would we configure the service version?

Reply:
It's already baked into the http pipeline instance here

Reply:
Wait, no it isn't.

Reply:
Even before this PR, we never actually used the service API version that was set in the client builder. Right now, our protocol layer has it hardcoded and has no way to set it. I'll touch base with Srikanta about this